addons/io/org.openhab.io.homekit/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Automatic-Module-Name: org.openhab.io.homekit
 Bundle-ClassPath: 
  .,
- lib/com.beowulfe.hap.hap-1.1.4.jar,
+ lib/hap-1.1.5.jar,
```

Comment:
Have you had a look at the new buildsystem?
The issue is, if I merge this PR, we add another 500kb of binary data to the repo (which is currently at 400 MB already) and with the buildsystem migration that need to happen anyway, we would render that binary data unused (git doesn't forget).

Reply:
@davidgraeff  I'll have a look; I'd much prefer to not commit this artifact to the repository. Anything you can point me to?

Reply:
https://bintray.com/bintray/jcenter/com.beowulfe.hap%3Ahap

Reply:
Is this upgrade needed to build? Otherwise we can upgrade the library easily with the migration PR.

Reply:
yes, it is

Reply:
If this is the only thing missing here, we should commit it. We have to do something about al those binaries in the source tree anyway, 500kB more or less make no difference IMO.